jumpstarter/states.py

```diff
@@ -65,20 +78,27 @@ def __init__(self, actor_state=ActorState):
                 actor_state.stopping.value.resources_released,
                 actor_state.stopping.value.dependencies_stopped,
             ],
-            trigger="stop",
+            trigger=ActorStateTriggers.stop,
             loop=False,
-            after="stop",
+            after=ActorStateTriggers.stop,
         )
 
         self.add_transition(
-            "stop", actor_state.stopping.value.dependencies_stopped, actor_state.stopped
+            ActorStateTriggers.stop,
+            actor_state.stopping.value.dependencies_stopped,
+            actor_state.stopped,
         )
 
         self.add_transition("report_error", "*", actor_state.crashed)
-        self.add_transition("start", actor_state.stopped, actor_state.starting)
+        self.add_transition(
+            ActorStateTriggers.start,
+            actor_state.stopped,
+            actor_state.starting,
+            after=ActorStateTriggers.start,
```

Comment:
So this is the only problem! I understand now.

Reply:
Yeah, we needed anothe `after=`